src/epd_driver/epd_driver.c

```diff
@@ -342,3 +377,84 @@ enum EpdDrawError epd_draw_image(EpdRect area, const uint8_t *data, const EpdWav
     };
     return epd_draw_base(area, data, no_crop, EPD_MODE_DEFAULT, temperature, NULL, waveform);
 }
+
+void epd_set_rotation(enum EpdRotation rotation) {
+      display_rotation = rotation;
+}
+
+uint8_t epd_get_rotation() {
+    return display_rotation;
+}
+
+uint16_t epd_rotated_display_width() {
+  uint16_t display_width = EPD_WIDTH;
+  switch (display_rotation) {
+          case EPD_ROT_PORTRAIT:
+              display_width = EPD_HEIGHT;
+          break;
+          case EPD_ROT_INVERTED_PORTRAIT:
+              display_width = EPD_HEIGHT;
+          break;
+          case EPD_ROT_INVERTED_LANDSCAPE:
+          case EPD_ROT_LANDSCAPE:
+          break;
+      }
+  return display_width;
+}
+
+uint16_t epd_rotated_display_height() {
```

Comment:
uint16_t -> int

Reply:
Updated in [last commit](https://github.com/martinberlin/epdiy-rotation/commit/bde584b9dd0dd0eb401f6a845eb9cafcdda8d744)